crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -1196,11 +1216,12 @@ export.was_procedure_called
 
     dup movdn.5
 
-    # get procedure info (PROC_ROOT, storage_offset, storage_size) from memory stored at index
-    exec.get_procedure_info
-    # => [MEM_PROC_ROOT, storage_offset, storage_size, PROC_ROOT, index]
+    # get procedure root from memory stored at index
```

Comment:
Nit: Would you mind adding the stack state after `dup movdn.5` just for consistency?